### PR TITLE
🌱  Automatically create namespace if its not found in hubaddon install command

### PR DESCRIPTION
**Pull request by RokibulHasan7**
[comment]: # ( Copyright Contributors to the Open Cluster Management project )

## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/cmd/install/hubaddon/exec.go [hunk 1]
```diff
@@ -77,6 +81,28 @@ func (o *Options) runWithClient() error {
...
+		if err != nil && errors.IsNotFound(err) {
+			ns, err = clientSet.CoreV1().Namespaces().Create(context.Background(), &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: o.values.Namespace,
+				},
+			}, metav1.CreateOptions{})
+			if err != nil {
+				return fmt.Errorf("failed to create namespace %s: %w", ns, err)
+			}
+
```
Comment (JustinKuli) [inline pkg/cmd/install/hubaddon/exec.go:101]: Sorry to have a formatting nitpick, but can you remove this newline?

**Review by JustinKuli — APPROVED** [review]
Other than the one nitpick, I think it looks great!

File: pkg/cmd/install/hubaddon/exec.go (hunk 1 above)
Comment (RokibulHasan7) [reply pkg/cmd/install/hubaddon/exec.go]: removed 😄

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks!

Adding additional reviewers for awareness and approval.

/cc @qiujian16 @zhujian7

**Comment by zhujian7** [comment]
LGTM

**Pull request merged**